src/validationinterface.h

```diff
@@ -33,7 +34,9 @@ void UnregisterAllValidationInterfaces();
 class CValidationInterface {
```

Comment:
> CValidationInterface has never listened to mempool, it was just an interface between the things in validation.cpp and other callers, now its also listening to mempool, putting it in the middle of the various callbacks we have going.

I guess I don't understand what "listening to mempool" means. CValidationInterface is a class with methods, and the methods seem to be called from the same places as before. I don't think it's an important point, I just don't get what this is trying to say.